attack.py

```diff
@@ -1,11 +1,10 @@
 # -*- coding: utf-8 -*-
 from argparse import ArgumentParser
 from concurrent.futures import ThreadPoolExecutor, as_completed
-from loguru import logger
+# from loguru import logger
 from os import system
 from pyuseragents import random as random_useragent
 from random import choice
-from requests.exceptions import ConnectionError

```

Comment:
No need to import and catch this error in an except block. You can see it down below